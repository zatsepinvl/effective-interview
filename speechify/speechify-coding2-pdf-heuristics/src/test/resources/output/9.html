            <html><body>
            <div style="display: flex; flex-direction: row">
            <div style="width: 50%">
            <h1>Expected</h1>
            <hr />
            <p><i>THE MUSCLE & STRENGTH PYRAMID: NUTRITION</i></p>
<p>identify where you fall within the spectrum of all of these values. To close it out, we will go over fiber intake recommendations for health and nutrient absorption.</p>
<h2>Level 3 â€” Micronutrients & Water Intake</h2>
<p>This section will go over the types of micronutrients and how to use your food choices to fulfill their daily requirements. We will then finish Level 3 with fluid intake recommendations and how to tell if you are consuming enough to keep yourself hydrated and functioning properly.</p>
<h2>MUSCLE AND STRENGTH NUTRITION PYRAMID</h2>
<p>05 SUPPLEMENTS</p>
<p>04 NUTRIENT TIMING</p>
<p>03 MICRONUTRIENTS</p>
<p>02 macronutrients</p>
<p>01 energy balance</p>
<p>BEHAVIOR AND LIFESTYLE</p>
<p><i>INTRODUCTION</i></p>
<p><i>23</i></p>
            </div>
            <div style="width: 50%">
            <h1>Actual</h1>
            <hr />
            
            </div>
            </div>
            </body></html>