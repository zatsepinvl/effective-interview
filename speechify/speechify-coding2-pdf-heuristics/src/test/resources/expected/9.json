[
  { "type": "Marginalia", "text": "THE MUSCLE & STRENGTH PYRAMID: NUTRITION" },
  {
    "type": "Paragraph",
    "text": "identify where you fall within the spectrum of all of these values. To close it out, we will go over fiber intake recommendations for health and nutrient absorption."
  },
  { "type": "Heading", "text": "Level 3 \u2014 Micronutrients & Water Intake" },
  {
    "type": "Paragraph",
    "text": "This section will go over the types of micronutrients and how to use your food choices to fulfill their daily requirements. We will then finish Level 3 with fluid intake recommendations and how to tell if you are consuming enough to keep yourself hydrated and functioning properly."
  },
  { "type": "Heading", "text": "MUSCLE AND STRENGTH NUTRITION PYRAMID" },
  { "type": "Paragraph", "text": "05 SUPPLEMENTS" },
  { "type": "Paragraph", "text": "04 NUTRIENT TIMING" },
  { "type": "Paragraph", "text": "03 MICRONUTRIENTS" },
  { "type": "Paragraph", "text": "02 macronutrients" },
  { "type": "Paragraph", "text": "01 energy balance" },
  { "type": "Paragraph", "text": "BEHAVIOR AND LIFESTYLE" },
  { "type": "Marginalia", "text": "INTRODUCTION" },
  { "type": "Marginalia", "text": "23" }
]
